xapian-bindings/configure.ac

```diff
@@ -11,6 +11,7 @@ dnl See xapian-core's HACKING document for details of the reasons for required
 dnl versions.
 AM_INIT_AUTOMAKE([1.13 parallel-tests -Wportability subdir-objects tar-ustar no-dist-gzip dist-xz])
 AC_CONFIG_SRCDIR([xapian-headers.i])
+AC_CONFIG_SRCDIR([xapianletor-headers.i])
```

Comment:
Don't do this.  `AC_CONFIG_SRCDIR` just needs to be invoked once, with a relative filename that should exist in the source tree but is unlikely to exist in a random other directory (so the macro can sanity check that we're in the right source tree or something like that).

Reply:
fixed in a fixup commit.